pom.xml

```diff
@@ -111,6 +110,11 @@
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-storage-partitioner</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-connect-storage-common-htrace-core4-shaded</artifactId>
+            <version>${project.version}</version>
```

Comment:
Adding the `project.version` here because in this branch and up to 10.0.x this connector's releases follow storage-common in lockstep. Will switch to `kafka.connect.storage.common.version` in 10.0.x